src/components/TaxPayer.tsx

```diff
@@ -1,21 +1,23 @@
-import React from 'react'
+import React, { ReactElement } from 'react'
 import { useForm } from 'react-hook-form'
 import { Box } from '@material-ui/core'
 import { useDispatch, useSelector } from 'react-redux'
-import { LabeledInput } from './labeledInput'
+import { LabeledInput, Patterns } from './labeledInput'
 import { saveTaxpayerInfo } from '../redux/actions'
+import { PagedFormProps } from './pager'
+import { TaxesState, TaxPayer } from '../redux/data'
 
-export default function TaxPayerInfo ({ navButtons, onAdvance }) {
+export default function TaxPayerInfo ({ navButtons, onAdvance }: PagedFormProps): ReactElement {
   const { register, handleSubmit, errors } = useForm()
   // const variable dispatch to allow use inside function
   const dispatch = useDispatch()
 
-  const prevFormData = useSelector(state => {
-    return state.information.taxpayer ?? {}
+  const prevFormData: TaxPayer | undefined = useSelector((state: TaxesState) => {
```

Comment:
this can be const prevFormData?: TaxPayer

Reply:
`const blah? = __` is not valid. That shorthand is just for interface properties, type declarations, function parameters.